bears/python/FutureBear.py

```diff
@@ -0,0 +1,29 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='futurize',
+        output_format='unified-diff',
+        result_message='The code could be futurized.',
+        diff_distance=0,
+        use_stdout=True,
+        use_stderr=False)
+class FutureBear:
```

Comment:
there's no documentationon what this actually does, how are our users supposed to know that without googling?